src/modules/esl-tooltip/core/esl-tooltip.ts

```diff
@@ -110,15 +116,10 @@ export class ESLTooltip extends ESLPopup {
     document.body.appendChild(this);
     super.onShow(params);
     this._updateActivatorState(true);
-
-    setTimeout(() => {
-      this.focus();
-    });
   }
 
   /** Actions to execute on hide Tooltip. */
```

Comment:
Comment sounds more correct like this: "Actions to execute after Tooltip hide." or "Actions to execute after hide of Tooltip"